tests/test_resize.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+import skimage
+from parameterized import parameterized
+
+from monai.transforms import Resize
+from tests.utils import NumpyImageTestCase2D
+
+
+class ResizeTest(NumpyImageTestCase2D):
+
+    @parameterized.expand([
+        ("invalid_order", "order", AssertionError)
```

Comment:
What's the "invalid_order" parameter used here?

Reply:
order must be an integer.